tests/integration/run-locally.sh

```diff
@@ -3,19 +3,22 @@
 set -e
 
 expandpath() {
-  CDPATH= cd -- "$1" && pwd -P
+  CDPATH=$(cd -- "$1" && pwd -P)
+  echo "$CDPATH"
 }
```

Comment:
No, this is unsetting `CDPATH`, which might be set by the user. We want to avoid using this in any case.